### PR TITLE
zh: revise access-dashboard.md

### PR DESCRIPTION
**Pull request by Yisaer**
### What is changed, added or deleted? (Required)
Update access-dashboard document

### Which TiDB Operator version(s) do your changes apply to? (Required)

- [x] master (the latest development version)
- [x] v1.1 (TiDB Operator 1.1 versions)
- [ ] v1.0 (TiDB Operator 1.0 versions)

### CODE REVIEW

File: zh/access-dashboard.md [hunk 1]
```diff
@@ -8,24 +8,39 @@ category: how-to
 
 TiDB Dashboard 是 TiDB 4.0 专门用来帮助观察与诊断整个 TiDB 集群的可视化面板，你可以在 [TiDB Dashboard](https://github.com/pingcap-incubator/tidb-dashboard) 了解详情。本篇文章将介绍如何在 Kubernetes 环境下访问 TiDB Dashboard。
 
+## 前置条件
+
+你需要使用 v1.1.1 版本及其以上的 TiDB Operator 以及 4.0.1 版本及其以上的 TiDB 集群才能在 Kubernetes 环境中流畅使用 `Dashboard`。 你需要在 `TidbCluster` 对象文件中通过以下方式开启 `Dashboard` 快捷访问:
```
Comment (ran-huang) [inline zh/access-dashboard.md:13]:
```suggestion
你需要使用 v1.1.1 版本及以上的 TiDB Operator 以及 4.0.1 版本及以上的 TiDB 集群，才能在 Kubernetes 环境中流畅使用 `Dashboard`。 你需要在 `TidbCluster` 对象文件中通过以下方式开启 `Dashboard` 快捷访问:
```

File: zh/access-dashboard.md [hunk 2]
```diff
@@ -8,24 +8,39 @@ category: how-to
...
+
```
Comment (ran-huang) [inline zh/access-dashboard.md:14]: @Yisaer We've already released v1.1.1?

Comment (Yisaer) [reply zh/access-dashboard.md:14]: No, but sooner.

**Pull request merged**